extensions/shared/block-icons.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import { isAtomicSite, isSimpleSite } from './site-type-utils';
+
+/**
+ * Returns the icon color for Jetpack blocks.
+ *
+ * Green in the Jetpack context, otherwise black for Simple sites or Atomic sites.
+ *
+ * @return {string} HEX color for block editor icons
+ */
+export function getIconColor() {
+	if ( isAtomicSite() || isSimpleSite() ) {
+		return '#1e1e1e'; // Default Gutenberg G2 Black
```

Comment:
Instead of hardcoding the color value, it would be nice if we could pull it from `@wordpress/base-styles` somehow, but I am not sure we can access those scss variables in a js file to begin with. We'll need to remember to keep that value updated. Maybe we could add a comment pointing to where the default Gutenberg black is defined, so we can refer to it and update ours when it changes?

Reply:
I looked this up and it seems like we'd need some sort of [custom webpack configuration](https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript) along with an `:export` command on the scss file, both of which I'm unsure how to make or if we can even make them.

Reply:
Added comments on https://github.com/Automattic/jetpack/pull/15311/commits/7633b59aab9c2d5f44d22679b0b79996f2ad0d1d